### PR TITLE
Combat2.0 chunk3. Attachments_ merge to superbranch

### PR DESCRIPTION
**Pull request by 1sourcecontrol**
@bleehu let me know what you think.

All the attachments are priced from... 5 months ago, or so when the weapons were also more expensive. So Idk if they should be lowered or not. I think having attachments not be dirt cheap creates more meaningful choice than the way it was before.

### CODE REVIEW

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 1]
```diff
@@ -1,63 +1,53 @@
...
-rectangular glass. Allows for smaller F.O.V. blockage. Has a small cross, 
-chevron, or dot as the indicator. Can only be used on pistols.
-Miss Chance -5%
-Range +5m
-
-1.2x Hologramic projector $175
+Reduction in Range Bracket Miss Chance by weapon type: -1 for Pistols, -0.5 other.
+R3 Range: +3m
+
+1x Reflexive Sight $500
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Shopuld be reflex site, not reflexive

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 2]
```diff
@@ -1,63 +1,53 @@
...
-1.2x Pistol Hologramic projector $125
-Projects a hologram target aquisition reticle, which compensates slightly
-for windage or target movement. also artificially enhances view so as to 
-magnify by 120%. Can only be used on pistols.
-Range +5m
-Miss Chance -10%
+magnify by 120%.
+R3 Range bonus & Range Bracket Miss Chance reductions are proportional to the weapon
+type: +8m/-2.0 Pistols, +12m/-1.5 submachine guns, +8m/-1.0 carbines, +6m/-1.0 other.
+Weapon Reflex: +5
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: What is weapon reflex?

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 3]
```diff
@@ -1,63 +1,53 @@
...
-Range +10m 
-
-1x Pistol Reflex Sight $100
-A reflex-projected site, comprised of a light source and a single pane of 
-rectangular glass. Allows for smaller F.O.V. blockage. Has a small cross, 
-chevron, or dot as the indicator. Can only be used on pistols.
-Miss Chance -5%
-Range +5m
-
-1.2x Hologramic projector $175
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: No more 1.2x site?

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 4]
```diff
@@ -1,63 +1,53 @@
...
+Reduction in Range Bracket Miss Chance by weapon type: -1 for Pistols, -0.5 other.
+R3 Range: +3m
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Only R3 increase? dot sights of all kinds should increase visibility primarily at lower ranges.

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt (hunk 1 above)
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Should be Reflex or maybe just give it a designation, as reflex is a style of sight not a trademark.

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 5]
```diff
@@ -1,63 +1,53 @@
...
-
-1x Pistol Reflex Sight $100
-A reflex-projected site, comprised of a light source and a single pane of 
-rectangular glass. Allows for smaller F.O.V. blockage. Has a small cross, 
-chevron, or dot as the indicator. Can only be used on pistols.
-Miss Chance -5%
-Range +5m
-
-1.2x Hologramic projector $175
+Reduction in Range Bracket Miss Chance by weapon type: -1 for Pistols, -0.5 other.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: "Reduction in all range brackets" maybe to make this a little more clear?

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 6]
```diff
@@ -33,71 +33,81 @@ for various lighting situations and terrains.
...
-Range + 50m, Minimum +10m.
+Add a range bracket with Miss Chance R4 = R3 + 1 and Range R4 = R3+35m, +10m minimum range.
+Range Bracket Miss Chance: -0.5
 
-4x Scope $350
-Range +75m, minimum +15m.
-Can only be used on Long Rifles, Carbines.
+4x Basic Scope  $1100
+Not usable on pistols, SMGs. 
+Add a range bracket with Miss Chance R4 = R3+1 and Range R4 = R3+50m, +15m minimum range.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Miss Chance R4 = R3 miss chance + 1;

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 7]
```diff
@@ -33,71 +33,81 @@ for various lighting situations and terrains.
...
+Range Bracket Miss Chance: -0.5
 
-2-7x Scope $700
-For 2 actions, you can adjust the magnification on this scope in order to 
+6x Basic Scope  $1400
+Not usable on pistols, SMGs. 
+Add a range bracket with Miss Chance R4 = R3+1 and Range R4 = R3+80m, +20m minimum range.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: See 4x comment

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 8]
```diff
@@ -33,71 +33,81 @@ for various lighting situations and terrains.
...
+
+2-6x Scope  $2100
+Not usable on pistols, SMGs. 
+For 1 action, you can adjust the magnification on this scope in order to 
 better see your target. Once sighted to a certain range, other targets that 
 are +/- 10 meters can still be fired upon. Otherwise you must adjust your 
 magnification.
-Miss Chance -20%
-Range +100m, minimum +10m
+Add a range bracket with Miss Chance R4 = R3+0.5 and Range R4 = R3+80m, +10 minimum range.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:82]: See 4x sight comment.

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 9]
```diff
@@ -33,71 +33,81 @@ for various lighting situations and terrains.
...
+Range Bracket Miss Chance: -0.5
 
-7x Scope $600
-Range +120m, minimum +30m
+7x Basic Scope  $1500
 Can only be used on Long Rifles.
+Add a range bracket with Miss Chance R4 = R3+1 and Range R4 = R3+50%m (or +95m whichever is higher), +25m minimum range.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:87]: Range R4 = 1.5x R3 / +95 w/e higher

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 10]
```diff
@@ -33,71 +33,81 @@ for various lighting situations and terrains.
...
+Range Bracket Miss Chance: -0.5
 
-8x Scope $750
-Range +150m, minimum +35m
+8x Basic Scope  $1800
 Can only be used on Long Rifles.
+Add a range bracket with Miss Chance R4 = R3+1 and Range R4 = R3+100%m, +30m minimum range.
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:92]: R4 = 2x R3

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 11]
```diff
@@ -107,75 +117,126 @@ type: +15m/-2.0 Pistols, +20m/-1.5 submachine guns, +15m/-1.0 carbines, +12m/-1.
 Minimum Range: +1m
 Weapon Reflex: +5
 
-====================
+4x Tritium-dot  $2000
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: 4x low-light site, can mentoin in notes powered by tritium

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 12]
```diff
@@ -107,75 +117,126 @@ type: +15m/-2.0 Pistols, +20m/-1.5 submachine guns, +15m/-1.0 carbines, +12m/-1.
...
-Only applies to balistic weapons. Not useable on coilguns, shotguns or HMGs.
+Auto and Burst Miss Chance: -0.5 
+R3 Range: +4m
+Only applies to ballistic weapons. Not useable on coilguns, shotguns, or Machine Guns.
+
+Improved Flash / Sound Suppressor  $1400
+Completely removes muzzle flash and the weapon report. Targets cannot locate 
+you due to firing your weapon.
+Auto and Burst Miss Chance: -0.5 
+R3 Range: +4m
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: +7/8m

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 13]
```diff
@@ -107,75 +117,126 @@ type: +15m/-2.0 Pistols, +20m/-1.5 submachine guns, +15m/-1.0 carbines, +12m/-1.
...
-====================
+========================================
 
-Light Profile Barrel $100
-Miss Chance +10%
-Str -1
+===== Level 1 Requirement =====
 
-Heavy Profile Barrel $100
+Light Profile Barrel  $300 + 10% weapon cost
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Prefer higher cost over scaling cost

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 14]
```diff
@@ -107,75 +117,126 @@ type: +15m/-2.0 Pistols, +20m/-1.5 submachine guns, +15m/-1.0 carbines, +12m/-1.
...
+Ranged Miss Chance: +0.5
+Strength Requirement: -1
+Carry Strength Bonus: +0.5 (cancels out movement speed penalties)
+Weapon Reflex: +5
+
+Heavy Profile Barrel  $450 + 10% weapon cost
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt]: Prefer higher cost over scaling cost

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 15]
```diff
@@ -213,210 +219,297 @@ Light Profile Barrel  $300 + 10% weapon cost
...
-Under Barrel Shotgun $910
-20m
-30%
-3 rds, 12ga Frangible
+CHEM: See Frontier Chemical Grenade.
+SMOKE: See ECSC Smoke Grenade.
+
+Under-Barrel Shotgun $520
+Damage, Ranged Miss Chance:			
+	130 damage, 2 miss chance out to 5m
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:295]: 110

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 16]
```diff
@@ -213,210 +219,297 @@ Light Profile Barrel  $300 + 10% weapon cost
...
-20m
-30%
-3 rds, 12ga Frangible
+CHEM: See Frontier Chemical Grenade.
+SMOKE: See ECSC Smoke Grenade.
+
+Under-Barrel Shotgun $520
+Damage, Ranged Miss Chance:			
+	130 damage, 2 miss chance out to 5m		
+	100 damage, 3 miss chance out to 10m
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:296]: 90

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 17]
```diff
@@ -213,210 +219,297 @@ Light Profile Barrel  $300 + 10% weapon cost
...
+Due to the positioning of the weapon, you can draw your primary as
+a free action. 
+Allows you to carry the gun on your back.
+Holstering your weapon is a free action.
+Strength Requirement: -1
+Range Bracket Miss Chance: -0.5
+
+===== Level 10 Requirement =====
+
+Pistol Smart-Lanyard System  $250
```
Comment (trowl223) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:373]: "Pistol Smart-Lanyard Holster System"

File: play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt [hunk 18]
```diff
@@ -1,359 +1,515 @@
...
-burst fire.
-See SMG rules for firing. Not useable with Revolvers.
-
-Pistol - Carbine kit $860
-A tactical frame bolts around the shooter''s pistol, and a longer barrel is 
+Pistol Hot-Load Reinforced Internals 
+	(Pistol) $1000 + 20% weapon cost
+	(Revolver) $1500 + 20% weapon cost
+These reinforced internals allow the pistol to fire heavier loads, increasing 
+damage output of the pistol. Not usable with revolvers or plasma pistols.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/04_02_WeaponAttachments.txt:453]: should be usable with revolvers

**Pull request merged**